samples/js/node/notebooks/read_mode_from_buffer.nnb

```diff
@@ -0,0 +1,166 @@
+{
+    "cells": [
+        {
+            "language": "markdown",
+            "source": [
+                "# Hello Image Classification\n\nThis basic introduction to OpenVINO™ shows how to do inference with an image classification model.\n\n A pre-trained [MobileNetV3 model](https://docs.openvino.ai/2023.0/omz_models_model_mobilenet_v3_small_1_0_224_tf.html) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/) is used in this tutorial. For more information about how OpenVINO IR models are created, refer to the [TensorFlow to OpenVINO](../101-tensorflow-to-openvino/101-tensorflow-to-openvino.ipynb) tutorial.\n   "
```

Comment:
This notebook is very similar to `001-hello-world.nnb`. The description should say what we want to present in this notebook and how it is different

Reply:
I removed this notebook and add tests